internal/worker/client/command/query.go

```diff
@@ -0,0 +1,40 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/renatoaguimaraes/job-scheduler/internal/worker/proto"
+	"google.golang.org/grpc"
+)
+
+type QueryCommand struct {
+	client proto.WorkerServiceClient
+	args   []string
+}
+
+func NewQueryCommand(client proto.WorkerServiceClient) Runner {
+	return &QueryCommand{
+		client: client,
+	}
+}
+
+func (c *QueryCommand) Init(args []string) {
+	c.args = args
+}
+
+func (c *QueryCommand) Run() {
+	ctx, cancel := context.WithTimeout(context.Background(), time.Millisecond*100)
```

Comment:
nit: this timeout is very aggressive, would cause problems when connecting to a server over a long distance or slow link

Reply:
It's true, was just for exercise purpose. It's a good candidate to be moved to the configuration.